focus-android/app/src/main/java/org/mozilla/focus/browser/integration/FindInPageIntegration.kt

```diff
@@ -39,10 +41,12 @@ class FindInPageIntegration(
 
     fun show(sessionState: SessionState) {
         findInPageView.isVisible = true
+        browserToolbar.isVisible = false
```

Comment:
Could we add a test for this? https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/test/java/org/mozilla/fenix/components/FindInPageIntegrationTest.kt